podcasts/PlayerContainerViewController.swift

```diff
@@ -267,6 +271,22 @@ class PlayerContainerViewController: SimpleNotificationsViewController, PlayerTa
     @objc func handleAppWillBecomeActive() {
         didSwitchToTab(index: tabsView.currentTab)
     }
+
+    // MARK: - Hide/Show tabs
+
+    func hideTabsAndLockScrollView() {
+        mainScrollView.isScrollEnabled = false
+        topSpaceToHeader.isActive = false
+        topSpaceToSafeArea.isActive = true
+        view.layoutIfNeeded()
+    }
+
+    func showTabsAndUnlockScrollView() {
+        mainScrollView.isScrollEnabled = true
+        topSpaceToHeader.isActive = true
+        topSpaceToSafeArea.isActive = false
+        view.layoutIfNeeded()
+    }
```

Comment:
🤔 Would it make sense to combine these or is that less clear?

```suggestion
    func updateTabsAndUnlockScrollView(isEnabled: Bool) {
            mainScrollView.isScrollEnabled = isEnabled
            topSpaceToHeader.isActive = isEnabled
            topSpaceToSafeArea.isActive = !isEnabled
            view.layoutIfNeeded()
    }
```

Reply:
Done in debf21cc39ffaf78438d7e377ef658ebccbcbb6a